stl/inc/experimental/generator

```diff
@@ -115,21 +113,19 @@ namespace experimental {
             using iterator_category = input_iterator_tag;
             using difference_type   = ptrdiff_t;
             using value_type        = _Ty;
-            using reference         = _Ty const&;
-            using pointer           = _Ty const*;
+            using reference         = const _Ty&;
+            using pointer           = const _Ty*;
 
             coroutine_handle<promise_type> _Coro = nullptr;
 
             iterator() = default;
-            iterator(nullptr_t) : _Coro(nullptr) {}
-
-            iterator(coroutine_handle<promise_type> _CoroArg) : _Coro(_CoroArg) {}
+            explicit iterator(coroutine_handle<promise_type> _CoroArg) noexcept : _Coro(_CoroArg) {}

```

Comment:
Pre-existing naming ultra-nitpick, `_CoroArg` is `_UglyCamel` but we generally use `_Ugly_snake` for variables. No change requested.

Reply:
Thanks - that's exactly the kind of unconventional usage I'm trying to correct here.